concent_api/core/tests/test_unit_authentication_utilities.py

```diff
@@ -105,7 +105,7 @@ def test_validate_golem_message_client_authorization_should_raise_400_error_when
 
 class ValidateGolemMessageSignedWithKeyUnitTest(TestCase):
 
-    def test_validate_golem_message_signed_with_key_should_not_raise_error_if_correct_message_and_key_is_used(self):
+    def test_handle_validating_if_list_of_golem_messages_is_signed_with_key_should_not_raise_error_if_correct_message_and_key_is_used(self):
```

Comment:
please rename also the test, when `handle_validating_if_list_of_golem_messages_is_signed_with_key` function name is changed

Reply:
As above, lets decide.

Reply:
I would be for renaming